autocti/extract/one_d/abstract.py

```diff
@@ -117,7 +152,11 @@ def stacked_array_1d_from(
         ).native
 
     def add_to_array(
-        self, new_array: aa.Array1D, array: aa.Array1D, pixels: Tuple[int, int]
+        self,
+        new_array: aa.Array1D,
+        array: aa.Array1D,
+        pixels: Optional[Tuple[int, int]] = None,
+        pixels_from_end: Optional[int] = None,

```

Comment:
I guess that decorator I mentioned in the other PR would also be applicable here